Library/Homebrew/os/mac.rb

```diff
@@ -36,6 +36,26 @@ def locate tool
       end
     end
 
+    def install_name_tool
+      if File.executable?(path = "#{Formula['cctools'].opt_prefix}/bin/install_name_tool")
+        Pathname.new(path)
+      else
+        locate(tool)
```

Comment:
There isn't a variable by this name - did you mean `locate("install_name_tool")`?